frontend/src/widgets/HorizontalTableWidget.scss

```diff
@@ -0,0 +1,76 @@
+@use '../colors.scss' as *;
+
+.smarthub-horizontal-table-widget {
+  height: calc(100% - 1.5rem);
+}
+
+.smarthub-horizontal-table-widget .usa-table-container--scrollable {
+  overflow-x: auto;
+  overflow-y: auto;
+}
+
+.smarthub-horizontal-table-widget .usa-table {
+  width: 100%;
+  height: 100%;
+}
+
+.smarthub-horizontal-table-widget .usa-table caption {
+  margin-bottom: 26px;
```

Comment:
can we use a standard margin here? margin-bottom-3 is 24, better not to introduce a random size